services/docsrs/docsrs.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
```

Comment:
This file should be named `docsrs.tester.js` in order for the test runner to pick it up